include/deal.II/distributed/tria_base.h

```diff
@@ -157,9 +177,26 @@ namespace parallel
      */
     struct NumberCache
     {
+      /**
+       * number of locally owned cells per MPI rank
```

Comment:
Can you make these comments complete sentences with a capital letter and a period at the end, while you're there?